submit/submit

```diff
@@ -248,7 +259,7 @@ drop-down box in the web interface.'''
         language_part = 'For LANGUAGE use the ID or a common extension.'
     else:
         language_part = 'For LANGUAGE use one of the following IDs or extensions:'
-        max_length = max([len(l['name']) for l in languages])
+        max_length = max([len(L['name']) for L in languages])
```

Comment:
Let's not use single capital letters for variables. If you think that a single lowercase `l` is confusing, then let's rename it to `lang` or so.